### PR TITLE
Show Offer name on Admin Page Issue 

### PR DESCRIPTION
**Pull request by ms-mgrant**
Copy of PR #270 by @k-granados without merge conflicts.

### CODE REVIEW

**Comment by santhoshb-msft** [comment]
image
There is data in the backup, subscription/offer/plan table,

File: deployment/Database/AMP-DB-5.1.sql [hunk 1]
```diff
@@ -385,6 +385,7 @@ CREATE TABLE [dbo].[Subscriptions](
 	[AMPQuantity] [int] NOT NULL,
 	[PurchaserEmail] [varchar](225) NULL,
 	[PurchaserTenantId] [uniqueidentifier] NULL,
+    [PlanGUId] [uniqueidentifier] NULL
```
Comment (alexdrenea) [inline deployment/Database/AMP-DB-5.1.sql:388]: Do we need this new field in the DB?

It doesn't look like it's needed for showing the offer name on the admin page

**Comment by santhoshb-msft** [comment]
@alexdrenea I will close this PR and open a new one.
To answer your question, yes we need this incase there are two planId with the sane name. Its hard to distinguish which planId the sub is referring to. Ideally we should have foreign key relationship between , so weighing up the options.

File: deployment/Database/AMP-DB-5.1.sql (hunk 1 above)
Comment (santhoshb-msft) [reply deployment/Database/AMP-DB-5.1.sql:388]: @alexdrenea I will close this PR and open a new one.
To answer your question, yes we need this incase there are two planId with the sane name. Its hard to distinguish which planId the sub is referring to. Ideally we should have foreign key relationship between , so weighing up the options.

**Comment by santhoshb-msft** [comment]
Will open as a new PR

**Pull request closed without merging**